### PR TITLE
Add translations property to be able to set translatable texts.

### LINKED ISSUE #45: Make it possible to provide all the text labels as a collection property
Currently to override labels you will need to override all the components

### PR DESCRIPTION
**Pull request by bubenkoff**
Closes #45

---
This change is [Reviewable](https://reviewable.io/reviews/sapientglobalmarkets/react-querybuilder/47)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -266,6 +266,45 @@ with the following properties:
 }
 ```
 
+#### translations *(Optional)*
+This can be used to override translatable texts applie to various controls
```
Comment (pavanpodila) [inline README.md]: typo here: "applied"

**Review by pavanpodila — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bubenkoff) [reply README.md]: done

**Comment by bubenkoff** [comment]
@pavanpodila please re-review

**Comment by bubenkoff** [comment]
@pavanpodila  thanks!
can you please make a new release?

**Comment by pavanpodila** [comment]
@bubenkoff Just published: https://www.npmjs.com/package/react-querybuilder

Thanks for helping out :-)

**Comment by pavanpodila** [comment]
Tweeted: https://twitter.com/pavanpodila/status/940255179311648768 @bubenkoff 

Feel free to retweet :-)

**Pull request merged**